features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/identity-event.properties

```diff
@@ -119,3 +119,9 @@ OIDCLogoutEventHandler.subscription.1=SESSION_TERMINATE
 OIDCLogoutEventHandler.enable=true
 module.name.21=tenantRegistrationVerificationHandler
 tenantRegistrationVerificationHandler.subscription.1=PRE_SET_USER_CLAIMS
+module.name.22=analyticsLoginDataPublisherV110
+analyticsLoginDataPublisherV110.subscription.1=AUTHENTICATION_STEP_SUCCESS
```

Comment:
You should add these configurations to the j2 file and default.json file as well